index.js

```diff
@@ -14,6 +23,13 @@ client.once(Events.ClientReady, readyClient => {
 	console.log(`Ready! Logged in as ${readyClient.user.tag}`);
 });
 
+client.on("messageCreate", msg => {
```

Comment:
We are using slash commands to interact with the bot. There's no need to create an event for `messageCreate`

Reply:
right